### ISSUE TITLE
Bulk moderation

### ISSUE THREAD

**Issue by Flexo013** [Flexo013/HeadsWebsite#15]
In the course of a few hours some person decided to upload tons of heads. Removing these one by one is quite a pain considering there are so many heads that can be deleted.

It would be awesome to have a separate page on which we can select multiple heads with tick boxes and then remove all selected heads by pressing one button.

Bunch of duplicates and broken heads:
![screenshot109](https://cloud.githubusercontent.com/assets/4507015/5241159/8c111612-7917-11e4-9d11-98be2d724bed.png)

**Comment by Fancypants** [comment]
Seems like a good enhancement but I don't think its much faster, instead of clicking the delete button you would be clicking a tick box (clicking the delete button removes the head almost instantly). 

Maybe it would be easier to select heads by clicking on their image (larger clickable area) so you would  spend less time aiming for a tick box or delete button. 

Unfortunately I'm a little tight on time, will look into it in when I have a little more time.

**Comment by Flexo013** [comment]
The main delay is caused my the reloading of the page in combination with the slight wait time for it to back to the scroll position.

Thanks for looking into it!

**Comment by Fancypants** [comment]
Created checkboxes for deleting heads. I separated renaming and deleting heads for easier coding :S, might change it in the future. Also we found that the database doesn't take up much space so instead of really deleting heads we put them into another database, currently not visible for mods, will change this. So if good heads accidentally get deleted we can still recover them. All heads, included already 'deleted' heads, take up as much as 1,6MB of space.
